### PR TITLE
[DOCS] Correct multi search API docs

### LINKED ISSUE #52477: Docs for `msearch` are incorrect
The [`msearch` documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-multi-search.html) lists a number of parameters under 'Request Body' when they should in fact be added either as path parameters or as the header for a subsearch.

### PR DESCRIPTION
**Pull request by jrodewig**
### Changes
* Adds an example request to the top of the page.
* Relocates several parameters erroneously listed under "Request body"
to the appropriate "Query parameters" section.
* Updates the "Request body" section to better document the NDJSON
  structure of msearch requests.

Closes #52477.

### CODE REVIEW

File: docs/reference/search/multi-search.asciidoc [hunk 1]
```diff
@@ -27,53 +37,219 @@ body\n
...
+
+`allow_no_indices`:::
+(Optional, boolean)
+If `true`, the request does *not* return an error if a wildcard expression or
+`_all` value retrieves only missing or closed indices.
++
+This parameter also applies to <<indices-aliases,index aliases>> that point to a
+missing or closed index.
+
+`expand_wildcards`:::
```
Comment (romseygeek) [inline docs/reference/search/multi-search.asciidoc]: Can this reuse the `common-parms` include as done for the request parameter above?

File: docs/reference/search/multi-search.asciidoc [hunk 2]
```diff
@@ -27,53 +37,219 @@ body\n
...
+included in the response. Defaults to `false`.
+
+`index`:::
+(Optional, string or array of strings)
+Index name or <<indices-aliases,alias>> used to limit the request. Wildcard
+expressions are supported. You can specify multiple indices as an array.
+
+`preference`:::
+(Optional, string)
+Node or shard used to perform the search.
```
Comment (romseygeek) [inline docs/reference/search/multi-search.asciidoc]: Can we specify the default value here?

File: docs/reference/search/multi-search.asciidoc [hunk 3]
```diff
@@ -27,53 +37,219 @@ body\n
...
+
+`request_cache`:::
+(Optional, boolean)
+If `true`, the request cache can be used for this search.
+See <<shard-request-cache>>.
```
Comment (romseygeek) [inline docs/reference/search/multi-search.asciidoc]: Default value?

**Review by romseygeek — APPROVED** [review]
Thanks for picking this up!  A couple of nits, but otherwise LGTM - no need for another review.

**Comment by jrodewig** [comment]
Thanks for your review @romseygeek.

**Comment by jrodewig** [comment]
### Backport commits
master 7f1d05c
7.x 98bcf06bae1
7.6 a173418f8f6

**Pull request merged**